website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -37,9 +37,15 @@ cd $HOME/projects/memcached-operator
 # so all API groups will be <group>.example.com
 operator-sdk init --domain example.com --repo github.com/example/memcached-operator
 ```
+API groups are a mechanism to group portions of the Kubernetes API. You're probably already familiar with
+some of the core Kubernetes API groups, such as `apps` or `rbac.authorization.k8s.io`. You should name your
+domain to group your resource types in meaningful group(s), both for ease of understanding and because these
+groups determine how access can be controlled to your resource types using RBAC. For more information, see [the core Kubernetes docs](https://kubernetes.io/docs/reference/using-api/#api-groups) and [the Kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/gvks.html).
```

Comment:
```suggestion
**Note:** Currently, the tool does not support work with multi-domains to create the APIs by default. Therefore, you will check below that when you use the subCommand `create api` you will not need to inform the domain anymore. You will only inform the group, the KInd and the Version that should be used. Therefore, all APIs owned by the project will be generated using the domain informed when you create the project.
```

Reply:
@jberkhahn I understand that the goal is to clarify that the APIs will be generated using the domain informed here, so I thought of something like the above. 

Also, note that the tool SHOULD be able to work/handle external APIs and we should be able to use create API to for example scaffold a controller for the core apis or apis created in another project which will have another domain. That is a limitation on the tool that we need to address see: https://github.com/kubernetes-sigs/kubebuilder/issues/1999

c/c @laxmikantbpandhare @asmacdo

Reply:
I don't think we should have a thing about a potential feature in a repo for another project linked in the Operator SDK docs. I think at most a blurb stating that multiple domains are not currently supported, but I don't know how I feel about explaining non-supported features, as it's not possible to enumerate everything we don't support.

Reply:
@jberkhahn,

What motivates you to want to add it to the docs?
Someone who would like to create APIs that have another domain
Someone, that wants to create controllers/api in the project to reconcile external/core types 

Am I right? What is the motivation to add this note in this doc? 

> I don't think we should have a thing about a potential feature in a repo for another project linked in the Operator SDK docs

SDK is Kubebuilder. SDK does not implement anything about what is scaffold by default. SDK only implement its helpers and specific features and we clarify it in other places as we link Kubebuilder docs/issues and etc in many other places.  

Therefore, it is an open source project and links the issue enables the community ( those that are looking for the feature) to contribute or vote and let us know how much that is or not important for them.

Reply:
To address the detailed info via FAQ I created an issue as discussed : https://github.com/operator-framework/operator-sdk/issues/5979